content/en/blog/2020/large-scale-security-policy-performance-tests/index.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Large Scale Security Policy Perfomance Tests
+subtitle: How does having security policies effect latency of requests
+description: How does having security policies effect latency of request.
+publishdate: 2020-09-02
+attribution: "Michael Eizaguirre (Google)"
+keywords: [test,security policy]
+---
+
+## Overview
+
+Istio has a wide range of Security policies which can be easily configured into systems of services. As the number of Security policies being applied increases it is important to understand how much latency these Security policies are adding to the system.
+
+This blog post goes over common Security policies use cases and how the number of Security policies or the number of specific rules in a Security policy can effect the overall latency of requests.
+
+## Setup
+
+There are a wide range of Security policies and many more combinations of those policies. We will go over 6 test cases that were determined to be more common/valuable to users.
+
+The following test cases are run in an environment which consists of a fortio client sending requests to a fortio server, except of that of a the baseline which has no Envoy sidecars deployed.
+{{< image width="55%" ratio="45.34%"
+    link="./istio_setup.svg"
```

Comment:
do not need to add `./` if `istio_setup.svg` is at the same level of your `index.md`. In you case, it is, so could be safe to remove `./` . Same for other places?

Reply:
Perfect updated